src/data/faq_de.json

```diff
@@ -146,6 +146,23 @@
                             "Unter dem Reiter Zertifikate kann die App nur '<b>EU Digitale</b> COVID-Zertifikate' scannen.",
                             "Siehe auch folgenden Eintrag zu <a href='#vac_cert_invalid' target='_self' rel='noopener'>ungültigen Impfzertifikaten</a>."
                          ]
+                    },
+                    {
+                        "title": "Wie lange ist mein Zertifikat gültig?​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand vom Juli 2021.",
+                            "Wir unterscheiden grundsätzlich​<ul><li>die Gültigkeit des digitalen Zertifikats (die <b>technische Gültigkeit</b>) und</li><li>die Gültigkeit des Impfschutzes (die <b>fachliche Gültigkeit</b>) im Sinne der 'Dauer des Schutzes'</li></ul>",

```

Comment:
This is somehow confusing because in the first paragraph the entry is very general, talking about all three forms of the DCC and now here it mentions that we differentiate between the technical validity and the vaccination protection. Many users without a vaccination certificate will ask himself e.g. what's the "fachliche Gültigkeit" of my test certificate.

Reply:
I agree ... I suggest to merge this article as it is for now, while we figure out how to cover test certificates and if test certificates have a fachliche validity at all

Reply:
Oki doke (;

Reply:
It's the same weakness as noted in the https://www.coronawarn.app/de/faq/ Glossary
"Gültigkeit eines EU digitalen COVID-Zertifikats"